lib/as_ocamlfind.ml

```diff
@@ -72,16 +74,20 @@ let parse_lines (byte, native, pp) =
     native_incs; native_objs; native_link;
     pp_incs; pp_objs; }
 
-let pkg_lookups ocamlfind name =
+let pkg_lookups ocamlfind native name =
```

Comment:
@samoht that's not what I was suggesting. I don't like the fact the `native` logic crept in `ocamlfind.ml` this is obfuscating what is going on and should be explicitely handled at the `unit` level where all the `native` conditioning is already done.